src/gtfs/write.rs

```diff
@@ -420,6 +420,7 @@ pub fn write_stop_times(
                     drop_off_type: st.drop_off_type,
                     local_zone_id: st.local_zone_id,
                     stop_headsign: stop_times_headsigns.get(&(vj_idx, st.sequence)).cloned(),
+                    timepoint: st.datetime_estimated,
```

Comment:
From http://gtfs.org/reference#stop_timestxt :
* ` ` (empty) = `1` = exact
* `0` = estimated

So here it would do the opposite for 0 and 1, no?

Reply:
nice catch mb